Jenkinsfile

```diff
@@ -61,6 +62,7 @@ pipeline {
                                 booleanParam('DCHECK_ALWAYS_ON', true)
                                 booleanParam('RUN_NETWORK_AUDIT', false)
                                 booleanParam('RUN_AUDIT_DEPS', false)
+                                booleanParam('RUN_UPSTREAM_TESTS', false)
```

Comment:
Shouldn't the defaults be different between run and skip?

Reply:
I don't think so. By default we want to run on Linux only. With SKIP we don't run at all and with RUN, we run on Windows. SKIP takes precedence over RUN.